components/join-us-modal/join-us-modal.jsx

```diff
@@ -0,0 +1,326 @@
+    this.refs[`step${this.state.currentStep}Form`].validates(valid => {
+      if (valid) {
+        if (this.state.currentStep === this.totalStep) {
+          // we've reached the last step of the form, submit the form
+          this.handleFormSubmit();
+        } else {
+          // more steps to complete
+          this.setState(this.getNextScreenState());
+        }
+      }
+    });
+  }
+
+  handleSkipBtnClick() {
+    let currentStep = this.state.currentStep;
+    let formValues = this.state.formValues;
+    let fieldsToReset = this.state.fields[`step${this.state.currentStep}`];
+
+    Object.keys(fieldsToReset).forEach(key => {
+      delete formValues[key];
+    });
+
+    this.setState(
+      Object.assign({ formValues }, this.getNextScreenState()),
+      () => {
+        if (currentStep === this.totalStep) {
+          this.handleFormSubmit();
+        }
+      }
+    );
+  }
+
+  renderProgressDots() {
+    let dots = [...Array(this.totalStep)].map((dot, i) => {
+      return (
+        <div
+          key={i}
+          className={`dot ${this.state.currentStep === i + 1 ? `filled` : ``}`}
+        />
+      );
+    });
+
+    return <div className="modal-progress">{dots}</div>;
+  }
+
+  renderSteps() {
+    return (
+      <div>
+        <Step
+          show={this.state.currentStep === 1}
+          heading="Complete your profile"
+          subhead="Please confirm your information to finish creating your profile."
+          hint="(This information came from your Google or GitHub account)"
+        >
+          <Form
+            ref="step1Form"
+            fields={this.state.fields[`step1`]}
+            inlineErrors={true}
+            onUpdate={(evt, name, field, value) =>
+              this.handleFormUpdate(evt, name, field, value)
+            }
+          />
+        </Step>
+        <Step
+          show={this.state.currentStep === 2}
+          heading="Add a short bio"
+          subhead="Tell other Network members a bit about yourself"
+        >
+          <Form
+            ref="step2Form"
+            fields={this.state.fields[`step2`]}
+            inlineErrors={true}
+            onUpdate={(evt, name, field, value) =>
+              this.handleFormUpdate(evt, name, field, value)
+            }
+          />
+        </Step>
+        <Step show={this.state.currentStep === 3} heading="Add a profile photo">
+          <Form
+            ref="step3Form"
+            fields={this.state.fields[`step3`]}
+            inlineErrors={true}
+            onUpdate={(evt, name, field, value) =>
+              this.handleFormUpdate(evt, name, field, value)
+            }
+          />
+        </Step>
+      </div>
+    );
+  }
+
+  renderConfirmation() {
+    return (
+      <div className="text-center">
+        <h2 className="h3-heading">Discover and Connect!</h2>
+        <p className="body-large">Thanks for signing up!</p>
+        <p>
+          Check out projects, connect with other members. Upload your projects
```

Comment:
was this supposed to be `[...] members, upload [...]`? As two separate sentences this text is a bit odd (both sentences right now list two things, but the first uses a comma, while the second uses an "or". That said, since it's really just a list of four things, we probably want one sentence that lists four things?)

Reply:
Thanks for catching this. I straight up copied and pasted the provided copy without thinking. Now I read it over I agree these sentences sound odd. @xmatthewx @natalieworth can you chime in here? We are talking about copy on this screen 

<img width="938" alt="image" src="https://user-images.githubusercontent.com/2896608/60685817-1cdcfa00-9e5a-11e9-8de6-48ac41fcae6e.png">

Reply:
per @natalieworth change it to:
Check out projects or upload your own, connect with other members, or add more to your profile.